raven/recipe/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: raven
+  version: {{ version }}
+
+source:
+  url: https://github.com/Zeitsperre/raven-feedstock/raw/add_ravenpy/raven/src/Raven-rev288.zip
```

I suppose this could be updated for now:
```suggestion
  url: https://github.com/CSHS-CWRA/raven-feedstock/raw/main/raven/src/Raven-rev288.zip
```